pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java

```diff
@@ -38,12 +36,9 @@
 @EqualsAndHashCode
 @ToString
 @Accessors(chain = true)
-public class HbaseAbstractConfig implements Serializable {
-
-    private static final long serialVersionUID = 6783394446906640112L;
+public class HbaseAbstractConfig {
 
     @FieldDoc(
-        required = false,
```

Comment:
:) why remove this `required` field?

Reply:
required = false is default value, can remove